### ISSUE TITLE
Calendar component is not responsive on smaller resolutions

### ISSUE THREAD

**Issue by AleksandrHovhannisyan** [ant-design/ant-design#25995]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://ant.design/components/calendar/](https://ant.design/components/calendar/)

### Steps to reproduce
1. Use the `Calendar` component.
2. Reduce the screen width.
3. Note that the table cells become miniscule on tablet and mobile.

### What is expected?
The component should be responsive. Otherwise, it's only useful on desktop.

### What is actually happening?
The calendar is shrinking indefinitely. The cells are too small to click.

| Environment | Info |
|---|---|
| antd | 4.5.2 |
| React | 16.6.0 |
| System | Windows 10 |
| Browser | Chrome 84.0.4147.105 |

**Comment by lihqi** [comment]
try [antd-mobile](https://mobile.ant.design/components/calendar/) on mobile

**Comment by Kermit-Xuan** [comment]
`antd` don't consider the response for mobile temporarily.

**Comment by AleksandrHovhannisyan** [comment]
I don't understand why this was closed. The component is not responsive even on tablets. The cells are quite small and you can barely fit any content in them without things getting cut off.

**Comment by Kermit-Xuan** [comment]
@AleksandrHovhannisyan 
The target device of `antd` is a personal computer, not a mobile device. The tablet is actually a mobile device. The main reasons are as follows:
On the one hand, responsive is not only responsive in style, but also functional compatibility. At present, our main energy is still on modern browsers such as Chrome, Firefox and Safari. If we want to be compatible with mobile devices, we may also need to consider mobile browsers such as Android and IOS.
On the other hand, most users do not only use the calendar component alone when using it, but also use it with other components. At present, other components also have responsive problems, so solving the responsive problem of the calendar component alone is only a symptom rather than a permanent solution. .
In summary, I suggest that you can use media queries and other means to solve your problems.
